doc/DEVELOPERS.md

```diff
@@ -64,9 +64,130 @@ make
 sudo make install
 ```
 
+### Configuration
+
+**If you have already set up the `myuser` and `repl` role, you can skip.**
+
+#### 1. Initialize PostgreSQL
+
+Open a new window, switch to the `postgres` user.
+
+``` sh
+sudo su -
+su - postgres
+```
+
+If you use dnf to install your postgresql, chances are the binary file is in `/usr/bin/`
+
+``` sh
+export PATH=/usr/bin:$PATH
+initdb /tmp/pgsql
+```
+
+Since we initialized the database in `/tmp`, the data in this directory might be removed after you go offline, depending on your OS configuration. If you want to make it permanent, choose a different directory.
+
+#### 2. Remove default acess
+
+Remove last lines from `/tmp/pgsql/pg_hba.conf`. The path mentioned can be found using the SQL command `SHOW hba_file;`.
+
+``` ini
+host    all             all             127.0.0.1/32            trust
+host    all             all             ::1/128                 trust
+host    replication     all             127.0.0.1/32            trust
+host    replication     all             ::1/128                 trust
+```
+
+#### 3. Add access for users and a database
+
+Add new lines to `/tmp/pgsql/pg_hba.conf`
+
+``` ini
+host    mydb             myuser          127.0.0.1/32            scram-sha-256
+host    mydb             myuser          ::1/128                 scram-sha-256
+host    postgres         repl            127.0.0.1/32            scram-sha-256
+host    postgres         repl            ::1/128                 scram-sha-256
+host    replication      repl            127.0.0.1/32            scram-sha-256
+host    replication      repl            ::1/128                 scram-sha-256
+```
+
+#### 4. Set password_encryption
+
+Set `password_encryption` value in `/tmp/pgsql/postgresql.conf` to be `scram-sha-256`
+
+``` sh
+password_encryption = scram-sha-256
+```
+
+For version 13, the default is `md5`, while for version 14 and above, it is `scram-sha-256`. Therefore, you should ensure that the value in `/tmp/pgsql/postgresql.conf` matches the value in `/tmp/pgsql/pg_hba.conf`.
+
+#### 5. Set replication level
+
+Set wal_level value in `/tmp/pgsql/postgresql.conf` to be `replica`
+
+``` sh
+wal_level = replica
+```
+
+#### 6. Start PostgreSQL
```

Comment:
We need to install the extension first

Reply:
I don't understand this part. 

Before configuring the `repl` role, we have already installed the `pgmoneta_ext` extension in the **Install pgmoneta_ext** section. 

This configuration is provided in case the user doesn't have the `repl` role.